src/draive/evaluators/coherence.py

```diff
@@ -71,12 +71,12 @@ async def coherence_evaluator(
         instruction=INSTRUCTION,
     )
 
-    if result := completion.extract_first(
-        "RESULT",
-        conversion=str,
+    if result := MultimodalTagElement.parse_first(
+        completion,
+        tag="RESULT",
     ):
         return EvaluationScore(
-            value=float(result) / 4,
+            value=float(result.content.as_string()) / 4,
```

Comment:
It can potentially crash if LLM's response can't be parsed as a float.

Reply:
it will raise an exception, it is expected, that is not a part of this PR